specification/semantic_conventions.md

```diff
@@ -108,6 +108,8 @@ instances. For each `LogRecord` instance:
 - `source.event.period` MUST contain the sampling period if this `LogRecord` represents a periodic event
 - `source.event.name` OPTIONALLY can contain the name of the event that triggered the sampling
 - `memory.allocated` MUST contain the allocation size if this `LogRecord` represents a memory allocation event
+- `com.splunk.profiling.data.type` MUST be either `allocation` or `cpu`
+- `com.splunk.profiling.data.format` MUST be either `text` or `pprof-gzip-base64`
```

Comment:
These names seem very Java-esq. Maybe instead:

```suggestion
- `profiling.data.type` MUST be either `allocation` or `cpu`
- `profiling.data.format` MUST be either `text` or `pprof-gzip-base64`
```

Also, these statements are normative only in value these fields must take on, but not whether these fields are to be included. Can These be updated to state if they should/must/may be included in payloads?

Reply:
changed `MUST be either` to ` MUST be set to either`